puppet/modules/koji/files/mirror_reposync.conf

```diff
@@ -0,0 +1,80 @@
+[main]
+reposdir=/dev/null
+
+[rhel-8-for-x86_64-baseos-rpms]
+name = Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
+baseurl = https://cdn.redhat.com/content/dist/rhel8/8/x86_64/baseos/os
+enabled = 1
+gpgcheck = 1
+gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+sslverify = 1
+sslcacert = /etc/rhsm/ca/redhat-uep.pem
+sslclientkey = /etc/pki/entitlement/3902935856580506602-key.pem
```

Comment:
Is this ID always the same or should it be a template with a parameter?

Reply:
I think it's the same per machine, so parameter it is